src/main/java/io/jenkins/plugins/forensics/miner/CommitStatistics.java

```diff
@@ -132,7 +132,7 @@ public String toString() {
 
     private static int getDistinctCount(final Collection<? extends CommitDiffItem> commits,
             final Function<CommitDiffItem, String> property) {
-        return (int) commits.stream().map(property).distinct().count();
+        return (int) commits.stream().map(p -> equalsIgnoreCase(property)).distinct().count();
```

Comment:
This will not work. You need something like:
```suggestion
        return (int) commits.stream().map(property).map(s -> String.toLowerCase(Locale.ENGLISH)).distinct().count();
```

Then the existing test should succeed.